plugins/performance-lab/readme.txt

```diff
@@ -15,13 +15,14 @@ The Performance Lab plugin is a collection of features focused on enhancing perf
 
 The feature plugins which are currently featured by this plugin are:
 
+* [Embed Optimizer](https://wordpress.org/plugins/embed-optimizer/)
 * [Image Placeholders](https://wordpress.org/plugins/dominant-color-images/)
+* [Image Prioritizer](https://wordpress.org/plugins/image-prioritizer/)
```

Comment:
I missed removing the `_(experimental)_` flag in https://github.com/WordPress/performance/pull/1846